metashare/api/tests/models.py

```diff
@@ -125,6 +139,7 @@ def test_full_org_type(self, user_factory, social_account_factory):
         user = user_factory(socialaccount_set=[])
         social_account_factory(
             user=user,
+            provider="salesforce-production",
```

Comment:
I see you're adding this a bunch of places; maybe it should be set as the factory's default?

Reply:
It's half the cases, actually; I'm gonna err on making `github` the default because it won't save me specification either way, and conceptually, a user should always have a github-provided social account, and only sometimes a salesforce-provided one.